pages/contact.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ContactPage from "../components/ContactPage";
+
+export default function contact() {
```

Comment:
Contact() with an uppercase C

Reply:
I have deleted the contact page in the pages because it is not currently used and this causes problems with vercel.